.storybook/preview.tsx

```diff
@@ -18,8 +18,9 @@ import {
 import { ComponentNameDecorator, RelatedComponentsDecorator } from "../storybook/components";
 import { DocsContainer, DocsPage, Unstyled } from "@storybook/blocks";
 import "vibe-storybook-components/index.css";
-import "../src/index.scss";
+import "../src/index.scss"; // needed only for colors story
```

Comment:
this comment is not clear, what do you mean?

Reply:
it's make sense since it's the storybook which is purpose is to display monday ui tokens, no?
```suggestion
import "../src/index.scss"; //  Importing color tokens, which are displayed in this story
```